socrates/lib/site/views/schedule.pug

```diff
@@ -117,7 +117,10 @@ block content
 
         dt Thursday
         dd.
-          The conference starts at 5pm on Thursday, August 24th 2017.
+          Check in should open at 4pm on Thursday, August 24th 2017. We will inform you
```

Comment:
Wenn ich lese "should xyz", höre ich immer in meinem Kopf "...but doesn't". Besser wäre nicht zu sagen, was passieren *sollte*, sondern was passieren *wird* imho